lib/active_model/serializer/configuration.rb

```diff
@@ -8,6 +8,7 @@ module Configuration
         base.config.array_serializer = ActiveModel::Serializer::ArraySerializer
         base.config.adapter = :attributes
         base.config.jsonapi_resource_type = :plural
+        base.config.type_source = :model
```

Comment:
The one I linked ;) https://github.com/rails-api/active_model_serializers/pull/1213/files#r40739506